.gitlab-ci.yml

```diff
@@ -92,14 +92,14 @@ variables:
   S3_DSD6_URI: s3://dsd6-staging
   RELEASE_VERSION_6: nightly
   RELEASE_VERSION_7: nightly-a7
-  DATADOG_AGENT_BUILDIMAGES: v3496931-201092e
-  DATADOG_AGENT_BUILDERS: v3399751-fc72c60
-  DATADOG_AGENT_WINBUILDIMAGES: v3496931-201092e
-  DATADOG_AGENT_ARMBUILDIMAGES: v3496931-201092e
-  DATADOG_AGENT_SYSPROBE_BUILDIMAGES: v3496931-201092e
+  DATADOG_AGENT_BUILDIMAGES: v3507010-02f01e5
+  DATADOG_AGENT_BUILDERS: v3508755-028d4c9
+  DATADOG_AGENT_WINBUILDIMAGES: v3507010-02f01e5
+  DATADOG_AGENT_ARMBUILDIMAGES: v3507010-02f01e5
+  DATADOG_AGENT_SYSPROBE_BUILDIMAGES: v3507010-02f01e5
   DATADOG_AGENT_LIBBCC_BUILDIMAGES: v2894686-d80c3ce
```

Comment:
I think this needs to stay pinned to the older image. Can anyone confirm?

Reply:
That's correct.